browser_tests/ComfyPage.ts

```diff
@@ -1066,7 +1066,7 @@ export class NodeReference {
     let clickPos: Position
     switch (position) {
       case 'title':
-        clickPos = { x: nodePos.x + nodeSize.width / 2, y: nodePos.y + 15 }
+        clickPos = { x: nodePos.x + nodeSize.width / 2, y: nodePos.y + 1 }
```

Comment:
Can you explain why this change is necessary? I think `+1` is clicking the node body not title. You can add a new position type `body` here if necessary.

Reply:
Yes, I was meaning to ask if 'title' meant the text or the title bar.

Using `+ 15` resulted in clicking the first-most widget of group nodes:

![Selection_464](https://github.com/user-attachments/assets/df65468a-4250-4286-9929-4d9f6f3d4254)

Reply:
I will create a body option

Reply:
I think the original value was `-15`?

Reply:
Seems to have been changed to `- 15` 3 hours ago in #1022